### PR TITLE
New imonomy Bid Adapter

### PR DESCRIPTION
**Pull request by ofir1122**
Adding support of imonomy prebid

test parameters for validating bids

      {
        bidder: 'imonomy',
        placementCode: 'foo',
        bidId: 'foo',
        sizes: [[300, 250]],
        params: {
          publisher_id: '14567721164',
        }
      }

contact email of the adapter’s maintainer
tech@imonomy.com
amit@imonmy.com
[X ] official adapter submission

## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'imonomy',
  params: {
           publisher_id: '14567721164'
  }
}
```

- contact email of the adapter’s maintainer
tech@imonomy.com
amit@imonmy.com
- [x] official adapter submission

### CODE REVIEW

**Comment by dbemiller** [comment]
Per the [project standards](https://github.com/prebid/Prebid.js/blob/master/CONTRIBUTING.md#contributing-to-prebidjs), this needs unit tests.

**Comment by ofir1122** [comment]
@dbemiller  Thank you for your comment. I added the unit tests.

File: modules/imonomyBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    for (var i = 0, l = adItems.length; i < l; i++) {
+      if(adItems[i].impression_id == imp ){
+        return adItems[i];
+      }
+    }
+
+    return null;
+  }
+
+  function encode64(input) {
```
Comment (dbemiller) [inline modules/imonomyBidAdapter.js]: Are you just re-implementing [atob](https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding) here? Or am I missing something?

**Comment by dbemiller** [comment]
I'm seeing a bid now, but no ad in the page.

The `integrationExamples/gpt/hello_world.hmtl` page exists to help test bid adapters end-to-end. Replace the AppNexus bid with your test params, run `gulp serve`, and then use that page to help debug.

**Comment by ofir1122** [comment]
@dbemiller hi seems that the bid response was under 10 cent. I configured it and now it works.

**Comment by jaiminpanchal27** [comment]
This is merged into master. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Pull request merged**